### DISCUSSION TITLE
Welcome to ElectionDapp Discussions!

### DISCUSSION THREAD

**Discussion by Shikhar15606** [Shikhar15606/ElectionDapp#126] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hammoudi00** [comment]
please i need help 
i need an example to .env file in this project 

PRIVATE_KEY = 
PROD_BLOCKCHAIN_URL =
ADDRESS =
MNEMONIC =
PORT = 5000
DB_URI =
STATS_DOC_ID =
JWT_SECRET =
COOKIE_SECRET =
NODE_ENV = development
SERVICE_ID =
ACCOUNT_SID = 
AUTH_TOKEN = 

Thank you in advance

**Reply by Shikhar15606** [reply]
Hi @hammoudi00 :wave: 
Here is the complete setup guide for this project [https://trapezoidal-calendula-968.notion.site/Setting-up-the-Project-0278e33f0c224ae4aea97b421e3240f3](https://trapezoidal-calendula-968.notion.site/Setting-up-the-Project-0278e33f0c224ae4aea97b421e3240f3)

Hope it helps :wink:
